src/create_input_qe.py

```diff
@@ -162,7 +162,7 @@ def print_occupations(occ):
 
 
 
-def write_qe_input(ex_st, label, mol, params,occ,occ_alp,occ_bet):
+def write_qe_input(ex_st, label, mol, params,occ,occ_alp,occ_bet,restart_flag):
```

Comment:
please, document all the variables that come into this function

Reply:
Done!